pkgs/by-name/wc/wcurl/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  lib,
+  curl,
+  fetchFromGitHub,
+  makeWrapper,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wcurl";
+  version = "2024.07.10";
+
+  src = fetchFromGitHub {
```

Comment:
Since it's a mirror from salsa.debian.org, we should directly use original source:

```nix
fetchFromGitLab {
  domain = "https://salsa.debian.org/debian/wcurl";
  owner = "debian";
  repo = "wcurl";
  # ...
};
```